contrib/babelfishpg_tsql/src/tsql_for/tsql_for.c

```diff
@@ -0,0 +1,73 @@
+/*-------------------------------------------------------------------------
+ *
+ * tsql_for.c
+ *   shared functions between forjson and forxml
+ *
+ *-------------------------------------------------------------------------
+ */
+#include "postgres.h"
+
+#include "tsql_for.h"
+
+/*
+ * This function handles the format for datetime datatypes by converting the output
+ * into required format for SELECT FOR JSON/XML. For example:
+ * "2022-11-11 20:56:22.41" -> "2022-11-11T20:56:22.41" for datetime, datetime2 & smalldatetime
+ */
+void
+tsql_for_datetime_format(StringInfo format_output, char *outputstr)
```

Comment:
Let's try to use `const char *` if we don't expect the input parameter be changed.

Reply:
Will do